### DISCUSSION TITLE
add_rule: aws_cloudfront_distribution with acm_certificate_arn not in us-east-1

### DISCUSSION THREAD

**Discussion by ineffyble** [terraform-linters/tflint-ruleset-aws#427] [Ideas]
CloudFront distributions can only be used with ACM certificates in us-east-1, which is a common gotcha.

It would be great (albeit a little tricky) if tflint could check if the certificate being passed to a CloudFront distribution is in the wrong region.

**Comment by wata727** [comment]
@ineffyble That's a good idea. A concern is that there is no way to get the currently configured region. There are several ways to configure region, so I think we need to access the aws-sdk-go internal API, but I'm not sure.

There is a way to access AWS Client from Runner, so it might be a good idea to extend this.
https://github.com/wata727/tflint/blob/c0787fdf290c8a8cf795a2aafbeae2a159801e64/tflint/runner.go#L29
